static/css/main.css

```diff
@@ -223,424 +225,422 @@ body {
+        font-size: 60px;
+        line-height: 60px;
+    }
+
+    .bgStatus .currentDetails {
+        font-size: 15px;
+    }
+
+    #silenceBtn * {
+      font-size: 20px;
+    }
+
+    .time {
+        padding-top: 0;
+        font-size: 20px !important;
+    }
+
+    .timebox {
+        width: auto;
+    }
+
+    #currentTime {
+        display: inline;
+        vertical-align: middle;
+    }
+
+    .timeOther {
+        display: inline;
+        margin-left: auto;
+        font-size: 15px;
+    }
+
+    #uploaderBattery label {
+        font-size: 15px !important;
+    }
+
+    .focus-range {
+        position: absolute;
+        top: 80px;
+        left: initial;
+        right: 10px;
+        margin: 0;
+        width: auto;
+    }
+
+    .focus-range li {
+        display: block;
+    }
+
+    #chartContainer {
+        top: 190px;
+    }
+    #chartContainer svg {
+        height: calc(100vh - (190px));
+    }
 }
 
 @media (max-height: 700px) {
-  #chartContainer {
-    font-size: 14px;
-  }
+    #chartContainer {
+        font-size: 14px;
+    }
 }
 
 @media (max-height: 600px) {
-  #chartContainer {
-    font-size: 12px;
-  }
+    #chartContainer {
+        font-size: 12px;
+    }
 }
 
 @media (max-height: 480px) {
-  #toolbar {
-    float: right;
-    height: auto;
-  }
+    #toolbar {
+        float: right;
+        height: auto !important;
+    }
 
-  #toolbar .customTitle {
-    display: none;
-  }
+    #toolbar h1 {
+        display: none;
+    }
 
-  .container {
-    top: 15px;
-    padding-bottom: 20px;
-  }
+    .container {
+        top: 15px;
+        padding-bottom: 20px;
+    }
 
-  #chartContainer {
-    font-size: 10px;
-  }
+    #chartContainer {
+        top: 130px;
+        font-size: 10px;
+    }
+    #chartContainer svg {
+        height: calc(100vh - 130px);
+    }
 }
 
 @media (max-height: 480px) and (min-width: 400px) {
-  .bgStatus .currentBG {
-    font-size: 70px;
-    line-height: 60px;
-  }
+    .bgStatus .currentBG {
+        font-size: 70px;
+        line-height: 60px;
+    }
 
-  .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
-    font-size: 70px;
-  }
+    .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
+        font-size: 70px;
+    }
 
-  .bgStatus .pill.direction {
-    font-size: 50px;
-    line-height: 50px;
-  }
+    .bgStatus .currentDirection {
+        font-size: 50px;
+        line-height: 50px;
+    }
 
-  .bgStatus .majorPills {
-    font-size: 15px;
-    line-height: 17px;
-  }
+    .bgStatus .currentDetails {
+        font-size: 15px;
+    }
 
-  .bgStatus .minorPills {
-    font-size: 12px;
-    line-height: 13px;
-  }
+    .time {
+        font-size: 50px;
+        line-height: 40px;
+        padding-top: 5px;
+    }
 
-  .status {
-    font-size: 50px;
-    padding-top: 5px;
-  }
+    .timeOther {
+        font-size: 15px;
+    }
 
-  .statusPills {
-    font-size: 15px;
-    line-height: 17px;
-  }
+    #uploaderBattery label {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293254/issues/source?bid=2452374&fileBranchId=2861014#l588)